internal/cephfs/controllerserver.go

```diff
@@ -167,11 +167,9 @@ func (cs *ControllerServer) CreateVolume(ctx context.Context, req *csi.CreateVol
 	if err != nil {
 		return nil, err
 	}
-	var ecip ErrCloneInProgress
-
 	vID, err := checkVolExists(ctx, volOptions, parentVol, pvID, sID, cr)
 	if err != nil {
-		if errors.As(err, &ecip) {
+		if errors.Is(err, ErrCloneInProgress) {
```

Comment:
this commit fixes up the previous introduction of the new error type. Please include these changes in the commits that introduce/consume that error type.

Reply:
Sure, Done.